manifests/nodejs.yml

```diff
@@ -849,13 +851,28 @@ tests/:
     Test_Config_IntegrationEnabled_False:
       '*': *ref_5_25_0
       express4-typescript: irrelevant
-      nextjs: irrelevant # nextjs is not related with kafka
+      nextjs: irrelevant
     Test_Config_IntegrationEnabled_True:
       '*': *ref_5_25_0
       express4-typescript: irrelevant
-      nextjs: irrelevant # nextjs is not related with kafka
+      nextjs: irrelevant
+    Test_Config_LogInjection_128Bit_TradeId_Default: missing_feature
+    Test_Config_LogInjection_128Bit_TradeId_Disabled:
+      '*': *ref_3_15_0
+      express4-typescript: irrelevant
+      nextjs: irrelevant
+    Test_Config_LogInjection_Default:
+      '*': *ref_3_0_0
+      express4-typescript: irrelevant
+      nextjs: irrelevant
+    Test_Config_LogInjection_Enabled:
+      '*': *ref_3_0_0
+      express4-typescript: irrelevant
+      nextjs: irrelevant
```

Comment:
are you sure those are irrelevant ? and not missing_feature instead ?

Reply:
Those apps just don't implement the route that tests the feature, but the feature exists

Reply:
irrelevant is for when something really makes no sense to test